build/build.proj

```diff
@@ -30,7 +82,7 @@
     </PropertyGroup>
 
     <Exec Command="$(NuGet) restore ..\src\Documentation\Documentation.sln" WorkingDirectory="$(MSBuildThisFileDirectory)" />
-    <Exec Command="(set SHFBROOT=$(SHFBROOT)) &amp; $(MSBuild) ..\src\Documentation\Documentation.sln /p:Configuration=$(Configuration) /nologo /v:minimal" WorkingDirectory="$(MSBuildThisFileDirectory)" />
+    <Exec Command="(set SHFBROOT=$(SHFBROOT)) &amp; $(MSBuild) ..\src\Documentation\Documentation.sln /p:Configuration=$(Configuration) /p:Platform=&quot;$(DocumentationPlatform)&quot; /nologo /v:minimal" WorkingDirectory="$(MSBuildThisFileDirectory)" />
```

Comment:
Please note that I had to add a separate platform configuration for the documentation project since it did not support the "Mixed Platforms" platform configuration.